docs/documentation/usage.rst

```diff
@@ -133,6 +133,30 @@ optimization problems. It is defined as in Python when assuming the OpenMDAO con
 
 For more details, please see the OpenMDAO documentation on `drivers <http://openmdao.org/twodocs/versions/latest/features/building_blocks/drivers/index.html>`_.
 
+Model options
+==============
+
+OpenMDAO 3.27 introduced a new way to set options for any component in the problem, using the
+:code:`model_options` attribute of the :code:`Problem` object (see OpenMDAO documentation `here <https://openmdao.org/newdocs/versions/latest/features/core_features/options/options.html#setting-options-throughout-a-problem-model-problem-model-options>`_).
+
+This can be controlled from FAST-OAD configuration file, using for instance:
```

Comment:
I'd suggest : "This can be controlled from the configuration file, using for instance"